release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2842,8 +2842,67 @@ This section will continue to be updated over time to provide notes on enhanceme
+
+* Previously, when you used the Agent-based Installer to install a cluster, `assisted-installer-controller` timed out or exited the installation process depending on whether `assisted-service` was unavailable on the rendezvous host. This situation caused the cluster installation to fail during CSR approval checks. With this release, an update to `assisted-installer-controller` ensures that the controller does not timeout or exit if `assisted-service` is unavailable. The CSR approval check now works as expected. (link:https://issues.redhat.com/browse/OCPBUGS-42839[*OCPBUGS-42839*])   
+
+* Previously, running the `openshift-install gather bootstrap --dir <workdir>` command might cause the installation program to skip analyzing analyze collected logs. The command would output the following message:
++
+[source,terminal]
+----
+Invalid log bundle or the bootstrap machine could not be reached and bootstrap logs were not collected
+----
++
+With this release, the installation program can now analyze log handles that the `gather bootstrap --dir <workdir>` argument generates. (link:https://issues.redhat.com/browse/OCPBUGS-42806[*OCPBUGS-42806*])
+
+* Previously, when you used the *Developer* perspective with custom editors on the {product-title} web console, entering the `n` keyboard shortcut caused the namespace menu to unexpectedly open. This issue happened because the keyboard shortcut key did not account for custom editors. With this release, the namespace menu now accounts for custom editors and does not unexpectedly open when you enter the `n` keyboard shortcut. (link:https://issues.redhat.com/browse/OCPBUGS-42607[*OCPBUGS-42607*])
+
+// Bug is marked as internal
+//* Previously, the installation program did not validate the maximum transmission unit (MTU) for a custom IPv6 network on {op-system-first}. If you specified a low value for the MTU, installation of the cluster would fail. With this release, the minimum MTU value for IPv6 networks is set to `1380`, where `1280` is the minimum MTU for IPv6 and `100` is the OVN-Kubernetes encapsulation overhead. With this release, the installation program now validates the MTU for a custom IPv6 network on {op-system-first} (link:https://issues.redhat.com/browse/OCPBUGS-41812[*OCPBUGS-41812*])
+
+* Previously, an {hcp} cluster that used mirroring release images might result in existing node pools to use the hosted cluster's operating system version instead of the `NodePool` version. With this release, a fix ensures that node pools use their own versions. (link:https://issues.redhat.com/browse/OCPBUGS-41552[*OCPBUGS-41552*])
```

Comment:
```suggestion
* Previously, a {hcp} cluster that used mirroring release images might result in existing node pools to use the hosted cluster's operating system version instead of the `NodePool` version. With this release, a fix ensures that node pools use their own versions. (link:https://issues.redhat.com/browse/OCPBUGS-41552[*OCPBUGS-41552*])
```